packages/core-graphql/__tests__/graphql.test.js

```diff
@@ -28,6 +27,79 @@ describe('GraphQL', () => {
   describe('test queries', () => {
     it('should be an object', () => {
       expect(testQueries).toBeObject()
+      logger.info('Processing test queries')
+    })
+
+    describe('test query for Block', () => {
+      it('should have queries', () => {
+        expect(testQueries.block.length).toBeTruthy()
+
+        logger.debug('Proceeding with Block test queries')
```

Comment:
The logger is a dependency of core and should not be used inside tests. Logging inside tests should be avoid in general.

Reply:
thanks, changing and pushing